packages/node_modules/@node-red/nodes/core/function/10-switch.html

```diff
@@ -176,14 +176,17 @@
                     for (var i=0;i<rules.length;i++) {
                         const opt = { label: RED._('node-red:switch.label.rule')+' '+(i+1) }
                         const r = rules[i];
-                        if (r.hasOwnProperty('v')) {
-                            if ((msg = RED.utils.validateTypedProperty(r.v,r.vt,opt)) !== true) {
-                                errors.push(msg)
+                        if (r.t !== 'istype') {
+                            if (r.hasOwnProperty('v')) {
+                                if ((msg = RED.utils.validateTypedProperty(r.v,r.vt,opt)) !== true) {
+                                    errors.push(msg)
+                                }
                             }
-                        }
-                        if (r.hasOwnProperty('v2')) {
-                            if ((msg = RED.utils.validateTypedProperty(r.v2,r.v2t,opt)) !== true) {
-                                errors.push(msg)
+                            if (r.hasOwnProperty('v2')) {
+                                if ((msg = RED.utils.validateTypedProperty(r.v2,r.v2t,opt)) !== true) {
+                                    console.log('HERE2', msg)
```

Comment:
I think this little guy shouldn't be here. 🙂

Reply:
Indeed not :)